### ISSUE TITLE
Accessibility - iOS - VoiceOver Does not Read Individual Steps on Main Steps Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#104]
While using VoiceOver to read the individual steps on the main Steps screen for English, they are not getting read.

**Comment by jfc3-dol** [comment]
On our, iPhone VoiceOver will only read the menu button and nothing less on the Main Steps page or any other screen for that matter. We are running iOS 11.1.2 on this device.

Our iPad is running iOS 11.2.1 (newer), and VoiceOver reads the eight steps as links but doesn't open them and go to the necessary screen as expected. Will create a new issue for this problem.

We haven't updated our operating systems to the newest (11.4.1) because not everyone does or can upgrade because it will cause other applications not to work.
